### PR TITLE
docs: document tweaking transaction isolation levels

### PR DESCRIPTION
**Pull request by ItayGarin**
Hey! I've added some docs for tweaking isolation levels.
Talked about this with @a8m on Discord.

### CODE REVIEW

File: doc/md/tutorial-todo-gql-tx-mutation.md [hunk 1]
```diff
@@ -50,6 +50,28 @@ func (mutationResolver) CreateTodo(ctx context.Context, todo TodoInput) (*ent.To
...
+	tx, err := c.client.BeginTx(ctx, &sql.TxOptions{Isolation: sql.LevelRepeatableRead})
+	if err != nil {
+		return nil, nil, err
+	}
+	ctx = ent.NewTxContext(ctx, tx)
+	ctx = ent.NewContext(ctx, tx.Client())
+	return ctx, tx, nil
+}
+
+srv.Use(entgql.Transactioner{TxOpener: &CustomTxOpener{client: client}})
```
Comment (a8m) [inline doc/md/tutorial-todo-gql-tx-mutation.md:72]: Can we use a simplified version using [TxOpenerFunc](https://github.com/ent/contrib/blob/master/entgql/transaction.go#L35)?
```suggestion
srv.Use(entgql.Transactioner{
	TxOpener: entgql.TxOpenerFunc(func(ctx context.Context) (context.Context, driver.Tx, error) {
		tx, err := client.BeginTx(ctx, &sql.TxOptions{Isolation: sql.LevelRepeatableRead})
		// ...
		return ctx, tx, nil
	})
})
```

**Review by a8m — APPROVED** [review]
Looks good. Just a minor suggestion.

**Comment by ItayGarin** [comment]
np. applied your suggestion @a8m

File: doc/md/tutorial-todo-gql-tx-mutation.md [hunk 2]
```diff
@@ -50,6 +50,20 @@ func (mutationResolver) CreateTodo(ctx context.Context, todo TodoInput) (*ent.To
 }
 ```
 
+## Isolation Levels
+
+If you'd like to tweak the transaction's isolation level. You can do so by implementing your own `TxOpener`. For Example -
```
Comment (a8m) [inline doc/md/tutorial-todo-gql-tx-mutation.md:55]:
```suggestion
If you'd like to tweak the transaction's isolation level, you can do so by implementing your own `TxOpener`. For example:
```

File: doc/md/tutorial-todo-gql-tx-mutation.md [hunk 3]
```diff
@@ -50,6 +50,20 @@ func (mutationResolver) CreateTodo(ctx context.Context, todo TodoInput) (*ent.To
...
 
+## Isolation Levels
+
+If you'd like to tweak the transaction's isolation level, you can do so by implementing your own `TxOpener`. For example:
+
+```go
+srv.Use(entgql.Transactioner{
+	TxOpener: entgql.TxOpenerFunc(func(ctx context.Context) (context.Context, driver.Tx, error) {
+		tx, err := client.BeginTx(ctx, &sql.TxOptions{Isolation: sql.LevelRepeatableRead})
+		// ...
```
Comment (a8m) [inline doc/md/tutorial-todo-gql-tx-mutation.md:61]:
```suggestion
		if err != nil {
			return nil, nil, err
		}
		ctx = NewTxContext(ctx, tx)
		ctx = NewContext(ctx, tx.Client())
```

**Pull request merged**